src/app/ui/utils/get-asset-contract-address.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * getAssetContractAddress
+ *
+ * Gets the assets contract address of a string: contract_id or fully qualified asset name.
+ *
+ * @param value - the source string: [principal].[contract-name] or [principal].[contract-name]::[asset-name]
+ */
+export const getAssetContractAddress = (value: string): string => {
+  if (value.includes('.')) {
+    return value.split('.')[0];
+  }
+
+  return value;
+};
```

Comment:
Can use fn declaration as we do elsewhere

Reply:
Also should have a unit tests

Reply:
fixed